samples/customer-registry-quickstart/README.md

```diff
@@ -0,0 +1,89 @@
+# Quickstart project: Customer Registry
+
+# TODO update for quickstart instructions
+
+## Designing
+
+While designing your service it is useful to read [designing services](https://developer.lightbend.com/docs/akka-serverless/designing/index.html)
+
+
+## Developing
+
+This project has a bare-bones skeleton service ready to go, but in order to adapt and
+extend it it may be useful to read up on [developing services](https://developer.lightbend.com/docs/akka-serverless/developing/index.html)
+and in particular the [Java section](https://developer.lightbend.com/docs/akka-serverless/java-services/index.html)
+
+
+## Building
+
+To build, at a minimum you need to generate and process sources, particularly when using an IDE.
```

Comment:
Not sure this sentence makes sense in the context of this sample, but I'm not sure how to change it?

Reply:
I agree this is a confusing sentence. Maybe just let's say "You can use Maven to build your project, which will also take care of generating code based on the `.proto` definitions:"?